### PR TITLE
Project developer tools for porting...

### PR DESCRIPTION
**Pull request by isabelheisters**

### CODE REVIEW

File: _bibliography/external/tool_pandt_project.bib [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+@inproceedings{KnuepferEtAl2012,
+  author       = {Knüpfer, A. and Rössel, C. and an Mey, D. and Biersdorff,
+    S. and Diethelm, K. and Eschweiler, D. and Geimer, M. and Gerndt, M. and Lorenz, D.
+    and Malony, A.D. and Nagel, W.E. and Oleynik, Y. and Philippen, P. and Saviankou, P.
+    and Schmidl, D. and Shende, S.S. and Tschüter, R. and Wagner, M. and Wesarg, B. and Wolf, F.},
+  booktitle     = {Tools for High Performance Computing 2011, Proceedings of the 5th
+    International Workshop on Parallel Tools for High Performance Computing, September 2011, Dresden, 
+    ed.: H. r Brunst, M.S. Müller, W.E. Nage, M.M. Resch, Springer, 2012. - 978-3-642-31475-9. -
+    S. 79 - 91},
+  cin          = {JSC},
```
Comment (torbjoernk) [inline _bibliography/external/tool_pandt_project.bib:87]: I know I'm nitpicking, but do you mind sticking to one single formatting style for BibTeX files?

File: _projects/tool_pandt_project.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+## Visits planned for the next 12 months
+
+Visit of RIKEN AICS researcher to JSC perhaps after ISC-HPC or in December 2015 (1–2
+weeks). Visit of JSC researcher to RIKEN considered for 2016 (perhaps 4 weeks).
+
+## Impact and publications
+
+The project just started and thus did not result in any publications, yet.
+
+{% bibliography --cited --file jlesc %}
```
Comment (torbjoernk) [inline _projects/tool_pandt_project.md]: Sorry, I advised you to drop the file extension, but this broke the LaTeX reporter. Just keep the file extension `.bib`.

File: _projects/tool_pandt_project.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+JUQUEEN BG/Q and Kei K computer. 
+Example batch scripts (e.g., to stage-in/out configuration and analysis files) and associated user 
+documentation will be provided.
+XcalableMP to be included in VI-HPS TuningWorkshop as part of PRACE Advanced Training
+Centre (PATC) curriculum to be held in Germany in spring 2016. 
+VI-HPS Tuning Workshop to be hosted by RIKEN AICS in Japan tentatively scheduled for February 2016.
+
+## References
+
+{% bibliography --file external/tool_pandt_project %}
```
Comment (torbjoernk) [inline _projects/tool_pandt_project.md]: Here as well.

**Comment by pancetta** [comment]
@torbjoernk Now what?

**Comment by torbjoernk** [comment]
Fixed with the sledge hammer. :hammer:

**Comment by pancetta** [comment]
@isabelheisters I got a Liquid Exception:

> Liquid Exception: PersonID 'wylie_b' not found. Typo? Otherwise add it to _data/people.yml. in _includes/asides/project_meta.html, included in _layouts/page_project.html

**Comment by torbjoernk** [comment]
Oh, @isabelheisters, we forgot to checkout `_data/people.yml` from the old branch we moved. You can do that.

**Comment by isabelheisters** [comment]
I will try

**Comment by torbjoernk** [comment]
And please check, before committing, that no other people got deleted. Best way is to use `git diff`.

**Pull request merged**